source/neuropod/tests/test_data/pytorch_strings_model/0/code/strings_model.py

```diff
@@ -5,7 +5,19 @@
 
 class StringsModel(nn.Module):
     def forward(self, x, y):
-        return {"out": np.array([f + " " + s for f, s in zip(x, y)])}
+        # Python 3 uses unicode for strings. Neuropod doesn't automatically convert
```

Comment:
Not sure the wording here is clear. Consider approaching from slightly different angle:
```
Neuropod string type are, similarly to Tensorflow and Pytorch, byte-arrays. A Python3 strings, are unicode strings. To convert a unicode string into a byte-array it needs to be encoded.
```
I think it would be clearer to use numpy encoding methods here instead of an ambiguous `x.astype` (unclear which encoding is used).

BTW, do we still support PY2? If yes, it would mess up a code a little I guess with conditions / six?